blockchain/src/blockchain/inherents.rs

```diff
@@ -31,20 +34,27 @@ impl Blockchain {
 
         inherents
     }
-    /// Given fork proofs and (or) a skip block, it returns the respective punishment inherents. It expects
-    /// verified fork proofs and (or) skip block.
+    /// Given equivocation proofs and (or) a skip block, it returns the respective punishment inherents. It expects
+    /// verified equivocation proofs and (or) skip block.
     pub fn create_punishment_inherents(
         &self,
         block_number: u32,
-        fork_proofs: &[ForkProof],
+        equivocation_proofs: &[EquivocationProof],
         skip_block_info: Option<SkipBlockInfo>,
         txn_option: Option<&db::TransactionProxy>,
     ) -> Vec<Inherent> {
         let mut inherents = vec![];
 
-        for fork_proof in fork_proofs {
-            trace!("Creating inherent from fork proof: {:?}", fork_proof);
-            inherents.push(self.inherent_from_fork_proof(block_number, fork_proof, txn_option));
+        for equivocation_proof in equivocation_proofs {
+            trace!(
+                "Creating inherent from equivocation proof: {:?}",
+                equivocation_proof,
```

Comment:
Use `tracing` style:
```suggestion
                ?equivocation_proof,
                "Creating inherent from equivocation proof",
```